### DISCUSSION TITLE
Update jQuery version

### DISCUSSION THREAD

**Discussion by sapnildessai** [ghiscoding/Angular-Slickgrid#881] [General]
Hi @ghiscoding ,
Just wanted to know if it is possible to upgrade jQuery version to 3.6.0 to keep it updated for security vulnerabilities. Our project scanner has scanned this library and reported that it is not using latest version of jQuery.

Thanks

**Comment by ghiscoding** [comment]
I will **not** until they release a new version and fix at the minimum this [issue](https://github.com/jquery/jquery/issues/4859), it's causing some of my Cypress E2E tests to fail in the Row Move (drag & drop). So I'm not interested to update for that reason, and the security issues aren't of concerns because I don't think SlickGrid wrote code in the way that they describe in the vulnerability mentioned.

**Reply by sapnildessai** [reply]
Sure thanks @ghiscoding . Have a nice day ahead :-)

**Reply by ghiscoding** [reply]
@sapnildessai just to let you know that the latest release [v4.2.1](https://github.com/ghiscoding/Angular-Slickgrid/releases/tag/v4.2.1) updated jQuery/jQueryUI to latest. This should fix all previous security issues identified. Cheers
